### PR TITLE
Add locations

### LINKED ISSUE #99: Add locations
# Description
The user should be able to define where a 'scheduled meal' is going to happen.
When creating an new 'scheduled meal' the user should see a dropdown with just two options:
- Amsterdam
- Rotterdam

The selection must be mandatory.

# Users
- Admin

=================================================================

Lunatech is growing and lunch has to be organised for more than one office on Fridays.
The current version does not have the notion off a location for the planned meals.

For admin users, add the possibility to add / modify locations.

The option to delete can also be added, but at least add a confirmation dialog to it.

### PR DESCRIPTION
**Pull request by arnieel**
Resolves #99

### CODE REVIEW

File: conf/evolutions/default/5.sql [hunk 1]
```diff
@@ -4,6 +4,7 @@ CREATE TABLE "MenuPerDay" (
   uuid UUID NOT NULL,
   "menuUuid" UUID NOT NULL,
   date Date NOT NULL,
+  location TEXT NOT NULL,
```
Comment (nMoncho) [inline conf/evolutions/default/5.sql]: This should be on a new Evolutions file (ie. 6.sql)

**Comment by arnieel** [comment]
If I understand correctly, you're asking if there's a validation for "hack" inputs for locations other than Amsterdam and Rotterdam? I didn't include any.

**Comment by nMoncho** [comment]
Yes, that's right. Maybe it's too much of a hassle, but would add some security over it (since we don't have foreign keys and such)

**Comment by arnieel** [comment]
I understand. Will add this.

**Comment by arnieel** [comment]
I added a Constants class for _OfficeLocations_ since 2 classes are using this. Please let me know if there's a better practice for declaring constants in Scala. :) Thanks!

File: conf/evolutions/default/11.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# --- !Ups
+
+ALTER TABLE "MenuPerDay" ADD COLUMN "location" TEXT NOT NULL;
```
Comment (LeonorLunatech) [inline conf/evolutions/default/11.sql]: You can't just add a non null column to an existing database. Remember that the DB in production already has data.
Or you define a default value or you create this in a few more steps (new column nullable, update column with data 'Rotterdam' and then modify it to be nullable).

**Pull request merged**